christfellowshipchurch/web-app-v2
# Updated `next`, `react`, and `stream-chat-react`

### About
In this PR I updated the following packages to their latest versions. After running the website with latest update I have not come across any bugs or anything. Also have noticed that almost all of the package version warnings in the console that were there before have left.

### Test Instructions
* delete `node_modules` and run `yarn`
* run `yarn dev` and check console for any errors
* website for any other issues.

### Note
Due to errors with the 'closeEmojiPicker' in chat for Groups, we will **remove update** for `stream-chat` until we have a proper solution.

### Closes Tickets
[CFDP-1683]


[CFDP-1683]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1683
